### PR TITLE
Fill in package.json

### LINKED ISSUE #22: Fill in package.json
I'll do this - `bantisocial`, version 0.0.1. GPL for the license (#19).

### PR DESCRIPTION
**Pull request by towerofnix**
Fixes #22.

Note the "homepage" field, which was automatically generated by `npm init` - I guess we're supposed to make a readme!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,27 @@
...
     "uuid": "^3.1.0"
   },
-
   "scripts": {
     "build": "parcel site/js/index.js site/script-nomodule.js -EM # see nathan/parcel-redux"
-  }
+  },
+  "name": "bantisocial",
+  "version": "0.0.1",
+  "description": "Actually free rip-off of Discord",
```
Comment (heyitsmeuralex) [inline package.json]: Nice

File: package.json [hunk 2]
```diff
@@ -8,8 +8,27 @@
...
+    "chat",
+    "chat-system",
+    "social"
+  ],
+  "author": "Some dummies who want more than 50 custom emotes",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/towerofnix/bantisocial/issues"
+  },
+  "homepage": "https://github.com/towerofnix/bantisocial#readme"
```
Comment (heyitsmeuralex) [inline package.json]: Could you add a readme of some kind in this PR? Dead links are bad!!

Comment (towerofnix) [reply package.json]: Yup. Made issue #26 (so this is blocked, for now).

**Comment by towerofnix** [comment]
@heyitsmeuralex Readme is here, so I think this can be merged now?

**Pull request merged**